app/templates/events/event_list.html

```diff
@@ -81,9 +81,9 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
     <tbody>
       {% for event in events %}
           {% if event.isPast %}
-            <tr class="table-light showlist" style="display: none;">
+            <tr height = '50' class="table-light showlist" style="display: none;">
```

Comment:
thank you for your help, Brian and Scott but unfortunately your coding experience has been trumped by the almighty and all-powerful god that is Zach Neil.

Reply:
Silly @zachneill . `d-none` is just the style tag for people who want don't want to feel dirty. also in this case they need the style because jquery's toggle() manipulates the style.